azure-pipelines.yml

```
@@ -241,3 +241,21 @@ jobs:
     - bash: |
         cargo check --all
       displayName: Check building with rust 1.40.0
+
+- job: PURE_RUST_POSTGRES
+  displayName: Test pure rust postgres connection implementation
```

I feel like this could be made more concise.

Maybe something like:
```suggestion
  displayName: Rust Postgres driver
```